tenets/codelingo/jenkinsx/tparallel/codelingo.yaml

```diff
@@ -7,13 +7,16 @@ tenets:
         body: You should NOT add t.Parallel() to an unencapsulated test as it may cause intermittent failures.
       codelingo/review:
         comment: You should NOT add t.Parallel() to an unencapsulated test as it may cause intermittent failures.
+      codelingo/rewrite:
+        name:
```

Comment:
name is empty?

Reply:
Ah you're trying to delete. I'll add a flag for that.

Reply:
I need to just remove. Should I do @rewrite "" instead of rewrite with empty string?